Sources/GRPC/Payload.swift

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright YEARS, gRPC Authors All rights reserved.
```

Comment:
This should be 2020 instead of YEARS (the script replaces acceptable years or ranges of years with "YEARS" since it's easier to check against a 'correct' header)